Test/Install-WhiskeyDotNetTool.Tests.ps1

```diff
@@ -19,6 +19,8 @@ function Init
     $script:globalDotNetDirectory = Join-Path -Path $TestDrive.FullName -ChildPath 'GlobalDotNetSDK'
     $script:version = $null
     $script:workingDirectory = $null
+    Mock -CommandName 'Get-Command' -ModuleName 'Whiskey' -ParameterFilter { $Name -eq 'dotnet' } -MockWith { $null }.GetNewClosure()

```

Comment:
You can actually just leave off the `-Mockwith` parameter and the mock will by default return `$null`.